README.md

```diff
@@ -98,23 +98,26 @@ Hierarchical child config via env, separated by using `__`. Specify list values
 | ENV                                                                    | DEFAULT VALUE | NOTE      |
 |------------------------------------------------------------------------|---------------|-----------|
 | [RUST_LOG](https://docs.rs/env_logger/latest/env_logger/) > LOG__LEVEL | "INFO"        | Log level |
-| SERVER__URL                                                            |               |           |
-| SERVER__PORT                                                           |               |           |
+| SERVER\_\_URL                                                          |               |           |
+| SERVER\_\_PORT                                                         |               |           |
 | SERVICE_NAME                                                           |               |           |
 | EXPORTER_ENDPOINT                                                      |               |           |
-| DB__PG__URL                                                            |               |           |
-| DB__PG__MAX_SIZE                                                       |               |           |
+| DB\_\_PG\_\_URL                                                        | "localhost"   |           |
+| DB\_\_PG\_\_MAX_SIZE                                                   | 5432          |           |
+| REDIS\_\_HOST                                                          | "localhost"   |           |
+| REDIS\_\_PORT                                                          | 6379          |           |
 
 Make sure to set these environment variables according to your needs before running the server.
 
 ## GitHub Flow CI Configuration
 
 1. **Set Docker Hub Secrets:**
-    - Go to repository Settings > Secrets.
-    - Add `DOCKER_USERNAME` and `DOCKERHUB_TOKEN`.
+
```

Comment:
Remove this line

Reply:
Formatted by **prettier**. It does not affect the file in view mode.